### DISCUSSION TITLE
Trying to add Apple Silicon (M1) releases to my project

### DISCUSSION THREAD

**Discussion by wimdeblauwe** [jreleaser/jreleaser#1587] [Q&A] [answered]
GitHub now has [M1 (Apple Silicon) based runners](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories) for open source projects. I am trying to incorporate this for my project that uses JReleaser.

I updated my build matrix to include `macos-14` which does the native build fine. 

I updated my JReleaser configuration in my `pom.xml` to include the extra artifact for the `osx-aarch_64` platform:

```
<artifact>
    <path>
        {{artifactsDir}}/{{distributionName}}-{{projectVersion}}-osx-aarch_64.zip
    </path>
    <platform>osx-aarch_64</platform>
</artifact>
```

However, [the release itself now fails](https://github.com/wimdeblauwe/ttcli/actions/runs/7746948997/job/21126672814) with:

```
distribution.ttcli.brew has more than one .zip artifact
```

See https://github.com/wimdeblauwe/ttcli/blob/main/pom.xml#L221 for my configuration.

The error message is not really clear to me, so any help is appreciated!

**Comment by aalmiray** [comment, marked as answer]
Default settings for `brew` packager require a single matching artifact. In this case there are 2 possibilities,`x86_64` and `aarch_64`, hence why it fails. There are a couple of ways to solve this:

- if single artifact is desired then you must select which platform, `x86_64` or `aarch_64`. Add `<extraProperties><skipBrew>true</skipBrew></extraProperties>` to the artifact that should not be chosen. https://jreleaser.org/guide/latest/reference/packagers/homebrew.html#_skip_artifacts
- switch the brew formula to multi-platform by adding `<multiPlatform>true</multiPlatform>` to the packager. No additional changes are required.

**Reply by wimdeblauwe** [reply]
I used `<multiPlatform>true</multiPlatform>` and the release is ok now. Thanks!
